tables/translated/kRO_english/hateffect_id_handle.txt

```diff
@@ -0,0 +1,155 @@
+1 HAT_EF_Blossom_Fluttering
```

Comment:
I think this is an extra file. We can use a common file: `tables\hateffect_id_handle.txt`